api/src/main/java/org/pmiops/workbench/api/SumoLogicController.java

```diff
@@ -50,22 +53,36 @@
   @Override
   public ResponseEntity<Void> logEgressEvent(String X_API_KEY, EgressEventRequest request) {
     authorizeRequest(X_API_KEY, request);
-
+    ObjectMapper mapper = new ObjectMapper();
+    EgressEvent[] events;
     try {
+      // Try to deserialize input with default ObjectMapper configuration. If failed to deserialize, log it then
+      // disable unknown properties restriction and try again.
       // The "eventsJsonArray" field is a JSON-formatted array of EgressEvent JSON objects. Parse
       // this out so we can work with each event as a model object.
-      ObjectMapper mapper = new ObjectMapper();
-      EgressEvent[] events = mapper.readValue(request.getEventsJsonArray(), EgressEvent[].class);
-      Arrays.stream(events).forEach(egressEventService::handleEvent);
-      return ResponseEntity.noContent().build();
-    } catch (IOException e) {
-      log.severe(
+      events = mapper.readValue(request.getEventsJsonArray(), EgressEvent[].class);
+    } catch (JsonProcessingException e) {
+      log.log(
+          Level.WARNING,
           String.format(
-              "Failed to parse SumoLogic egress event JSON: %s", request.getEventsJsonArray()));
-      log.severe(e.getMessage());
-      this.egressEventAuditor.fireFailedToParseEgressEventRequest(request);
-      throw new BadRequestException("Error parsing event details");
+              "Failed to parse SumoLogic egress event JSON: %s. Disable unknown properties restriction and try again...",
```

Comment:
nit: when I read this, it sounds like instructions to someone who is debugging - probably would slightly change the wording to "Disabling unknown properties restriction and trying again...", to indicate the code is doing this automatically